src/main/java/com/cinema/Application.java

```diff
@@ -68,11 +70,9 @@ public static void main(String[] args) throws AuthenticationException {
         shoppingCartService.addSession(session,roma);
```

Comment:
let's register() users instead of calling `userService.add()` and then `shoppingCartService.registerNewShoppingCart()`